gslab_scons/_release_tools.py

```diff
@@ -238,6 +238,70 @@ def up_to_date(mode = 'scons', directory = '.'):
     return bool(result)
 
 
+def list_ignored_files(dir_path = './release'):
+    '''List files ignored by git 
+
+    This function returns a list of the files in the directory
+    given by the dir_path argument that are ignored by git. 
```

Comment:
I'm not sure it's really necessary for this function to restrict attention to the files in a single directory. I think it would suffice to just get a normalized list of all files that are ignored and then if a file is in this ignore list, we don't issue a size warning.